SimPEG/EM/TDEM/SrcTDEM.py

```diff
@@ -86,6 +91,7 @@ class BaseTDEMSrc(BaseEMSrc):
 
     waveformPair = BaseWaveform  #: type of waveform to pair with
     waveform = None  #: source waveform
+    SrcType = "Inductive"
```

Comment:
This was how I am deciding whether computing initial condition or not. 
I can set that to None as a default